docs/recipes/svelte.md

```diff
@@ -0,0 +1,69 @@
+# Usage with Svelte
+
+Xstate integrates great with Svelte. (Especially Svelte stores!)
+
+The only gotcha is that you'll need this extra package.
+```js
+npm install @rollup/plugin-replace --save-dev
+```
+Then add this to the plugins array in rollup.config.js.
+```js
+replace({ 
+    'process.browser': true, 
```

Comment:
is this one needed? I don't think we use `process.browser` anywhere

Reply:
https://svelte.dev/repl/e36fa239641a49509ca1d06bcebf9f36?version=3.21.0

Certainly complains about it in the svelte repl.

Reply:
It complains about `process` being undefined but this happens because `process.env.NODE_ENV` is left (on purpose) in the distributed files. My question was purely about `process.browser` though - I'm not aware of such a thing being used by us.

Reply:
oh yeah should add in here that i use npm i xstate to install. 
With the CDN tag, i don't need that plugin.

Hmm I will try to look into why this is happening. I was told this error has to do with a package meant for node being used on the client